README.md

```diff
@@ -9,10 +9,10 @@ This file could place the ept cache on the globe.
 ## How convert to ept cache
 
 * Install :
-	* `Node js`
-	* `parallel`
-	* `Pdal`
-	* `entwine`
+	* `Node js` (https://nodejs.org or via package managers)
```

Comment:
Could you link with install page ? (by example https://pdal.io/download.html#conda)

and preferred installations (by example conda to install pdal)